presto-hive/src/main/java/com/facebook/presto/hive/HivePartialAggregationPushdown.java

```diff
@@ -263,7 +263,8 @@ private boolean isAggregationPushdownSupported(AggregationNode partialAggregatio
                     oldTableLayoutHandle.isPushdownFilterEnabled(),
                     oldTableLayoutHandle.getLayoutString(),
                     oldTableLayoutHandle.getRequestedColumns(),
-                    true);
+                    true,
+                    false);
```

Comment:
why are we passing in false for the appendRowNumber here. If this should be false, should we checkState oldTableLayoutHandle.getAppendRowNumber() is false.

Reply:
Good catch, it should pass oldTableLayoutHandle.isAppendRowNumber()